pkg/validation/options.go

```diff
@@ -41,7 +41,9 @@ func Validate(o *options.Options) error {
 		if err == nil {
 			transport := &http.Transport{
 				TLSClientConfig: &tls.Config{
-					RootCAs: pool,
+					RootCAs:    pool,
+					MinVersion: tls.VersionTLS12,
```

Comment:
Let's split this change into its own PR.  This is definitely a breaking change & warrants its own entry in the CHANGELOG.

I'll need to review what the EOL schedule of popular browsers is for TLS 1.0 & 1.1 (in particular browsers that are in the corporate/enterprise space since they seem to lag on browser updates compared to individuals and they are our userbase).